vms/platformvm/api/static_service.go

```diff
@@ -98,6 +98,17 @@ type Staker struct {
 	StakeAmount *json.Uint64 `json:"stakeAmount,omitempty"`
 }
 
+// GenesisStaker is very similar to Staker, but it explicitly uses a ShortNodeID
+// GenesisStaker should be used to generate genesis content only.
+// APIs should use Staker struct only
+type GenesisStaker struct {
+	TxID        ids.ID          `json:"txID"`
+	StartTime   json.Uint64     `json:"startTime"`
+	EndTime     json.Uint64     `json:"endTime"`
+	Weight      json.Uint64     `json:"weight"`
+	ShortNodeID ids.ShortNodeID `json:"nodeID"`
```

Comment:
Is there a reason to call this `ShortNodeID`?
```suggestion
	NodeID ids.ShortNodeID `json:"nodeID"`
```

Reply:
Done